test/unit/BCoWPool.t.sol

```diff
@@ -130,3 +148,84 @@ contract BCoWPool_Unit_EnableTrading is BaseCoWPoolTest {
     bCoWPool.enableTrading(appData);
   }
 }
+
+contract BCoWPool_Unit_Verify is BaseCoWPoolTest {
+  function test_Revert_NonBoundToken() public {
+    GPv2Order.Data memory order = correctOrder;
+    order.buyToken = IERC20(makeAddr('other token'));
+    order = correctOrder;
```

Comment:
I dont understand this part. Saving the buyToken and then overriding the entire order?

Reply:
I used to have one testcase for each non-bound token and merged them since they both tested the same line. I fat-fingered that one pretty bad :sob: